### PR TITLE
chore: Upgrade Capsule Dependencies to Latest Versions

### PR DESCRIPTION
**Pull request by jlm0**
### Description
This PR updates the dependencies for the @web3-onboard/capsule package to the latest versions:

Upgraded @usecapsule/react-sdk from ^2.3.1 to ^3.7.0
Upgraded @usecapsule/wagmi-v2-integration from ^1.7.1 to ^2.8.0

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [ ] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [ ] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [ ] Add/update the appropriate package README (if applicable)
- [ ] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [ ] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

### CODE REVIEW

**Comment by jlm0** [comment]
@Adamj1232 Hey Adam! Version upgrade for Capsule modal to support our latest features. Can you provide a review? Thanks!

File: packages/capsule/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/capsule",
-  "version": "2.1.0",
+  "version": "3.0.0-alpha.1",
```
Comment (Adamj1232) [inline packages/capsule/package.json:3]:
```suggestion
  "version": "2.2.0-alpha.1",
```
As I dont see a breaking change we can just bump with a minor version

**Comment by Adamj1232** [comment]
@jlm0 thanks for this! Couple small notes but looking good

**Comment by jlm0** [comment]
@Adamj1232 All done! Thanks!

**Review by Adamj1232 — APPROVED** [review]
Looks great thanks @jlm0 !

**Pull request merged**